### DISCUSSION TITLE
for 1.19 plz!

### DISCUSSION THREAD

**Discussion by Baeritto21** [GriefPrevention/GriefPrevention#1889] [Ideas]
Can yall make this plugin for 1.19 please?im running a 1.19 server with my friends and I would really like having this on that server but its not on 1.19!!!

**Comment by Jikoo** [comment]
What is not working on 1.19? The only new grief behaviors are using water bottles to create mud and using allays to steal items. The former should already be blocked by the same handling that takes care of stripping logs and pathing dirt, and the latter is so niche that I really doubt it would be an issue.
